include/rtc/configuration.hpp

```diff
@@ -27,12 +27,22 @@
 namespace rtc {
 
 struct IceServer {
+	enum class Type { STUN, TURN };
```

Comment:
Since I use a format like `enum class Type { Stun, Turn }` everywhere else, could you change it for the sake of consistency ?

Reply:
Sure